core/cache/src/test/java/uk/gov/gchq/gaffer/cache/CacheTest.java

```diff
@@ -76,8 +68,8 @@ public void shouldAddAndGetCacheNoOverwrite() throws CacheOperationException {
     }
 
     @Test
-    public void shouldGetCacheServiceName() {
-        assertEquals("serviceName1", cache.getCacheName());
+    public void shouldGetCacheName() {
+        assertEquals("cacheName1", cache.getCacheName());
```

Comment:
Elsewhere you have updated tests from junit to assertJ but you haven't done so in this file. Since you have changed these tests it might make sense to update them for consistency.

Reply:
I've now done this